fe/fe-core/src/main/java/com/starrocks/qe/Coordinator.java

```diff
@@ -1544,6 +1544,7 @@ public void updateFragmentExecStatus(TReportExecStatusParams params) {
         // and returned_all_results_ is true.
         // (UpdateStatus() initiates cancellation, if it hasn't already been initiated)
         if (!(returnedAllResults && status.isCancelled()) && !status.ok()) {
+            ConnectContext.get().setErrorCode(status.getErrorCode().toString());
```

Comment:
Is it possible that this errorCode be `null`?

Reply:
ok, i really should add judgment logic here